modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -267,6 +271,18 @@ void WebBrowserProcessor::process() {
         browser_->GetMainFrame()->ExecuteJavaScript(js_.get(), "", 1);
     }
 
+    if (dataFrames_.isReady()) {
+        auto frame = browser_->GetMainFrame();
+        size_t dataFrameNr = 0;
+        for (const auto& dataFrame : dataFrames_) {
+            nlohmann::json jsonString = *dataFrame;
+            std::stringstream data("var data = ", std::ios_base::app | std::ios_base::out);
+            data << jsonString.dump();
+            data << ";onInviwoDataChanged" << dataFrameNr++ << "(data);";
```

Comment:
Better to have the number as argument: onDataFrameChanged(json, dataFrameNr) ?

Reply:
Sure!